src/editor/components/Editor.tsx

```diff
@@ -10,20 +10,49 @@ import {
   materialRenderers,
 } from '@jsonforms/material-renderers';
 import { JsonForms } from '@jsonforms/react';
+import { Typography } from '@material-ui/core';
 import React from 'react';
+import { useDrop } from 'react-dnd';
 
-import { useUiSchema } from '../../core/context';
+import { useDispatch, useUiSchema } from '../../core/context';
+import { UI_SCHEMA_ELEMENT } from '../../core/dnd';
+import { Actions } from '../../core/model';
 import {
   DroppableHorizontalLayoutRegistration,
   DroppableVerticalLayoutRegistration,
 } from '../../core/renderers/DroppableLayout';
 import { useExportSchema } from '../../core/util/hooks';
-
+const NoUISchemaComponent: React.FC = () => {
+  const dispatch = useDispatch();
+  const [{ isOver, uiSchemaElement }, drop] = useDrop({
+    accept: UI_SCHEMA_ELEMENT,
```

Comment:
I like that you also added an "initial" behavior. For generality it would be nice if we would also allow a ui schema which only consists of a single control as this is a valid use case. However this does not have to be done in this PR.